plugins/org.jboss.tools.openshift.common.ui/build.properties

```diff
@@ -2,4 +2,7 @@ source.. = src/
 output.. = bin/
 bin.includes = META-INF/,\
                .,\
-               plugin.xml
+               plugin.xml,\
+               plugin.properties,\
+               icons/,\
+               pom.xml
```

Comment:
@adietish fixing.  side-effect of some other changes...must have been trigger happy